src/OpenTelemetry.Instrumentation.Wcf/WcfInstrumentationOptions.cs

```diff
@@ -48,4 +48,17 @@ public class WcfInstrumentationOptions
     /// Gets or sets a value indicating whether or not the SOAP message version should be added as the <see cref="WcfInstrumentationConstants.SoapMessageVersionTag"/> tag. Default value: False.
     /// </summary>
     public bool SetSoapMessageVersion { get; set; }
+
+    /// <summary>
+    /// Gets or sets a value indicating whether exception will be recorded
+    /// as an <see cref="ActivityEvent"/> or not. Default value: <see
+    /// langword="false"/>.
+    /// </summary>
+    /// <remarks>
+    /// <para><b>RecordException is supported on all runtimes.</b></para>
```

Comment:
You probably don't need this note about runtime compatibility. My guess is you copied this from HttpClient project where it is meaningful because not all options apply to all runtimes.